app/controllers/admin_events_controller.rb

```diff
@@ -3,6 +3,7 @@
 class AdminEventsController < ApplicationController
   before_action :get_event, except: [:index, :annual_events_report]
   before_action :require_admin
+  before_action :get_approved_tickets_count, only: [:show]
```

Comment:
@guainumbi Is this strictly necessary? If yes, I don't understand the use. Could you please explain?

Reply:
This is updating the database with the approved_tickets_count of an event (when event is created the value is nil at first, the first view of the events admin_show view updates the db with current count. from then on we add +1 to the count whenever we approve an application). I guess we can also call the method in the show view, but I thought it is cleaner like this. Another solution would be to have the default value of approved_ticket_count as 0...

Reply:
I think we should have a default value in the db better, but I will leave you to your choice. What about the UI comment I left on top of this comment? :) Are you thinking about modifying it?

Reply:
changing to integer with default value. yes, i think it makes sense to think about different layout.